reviews-receiver/src/routes/batch.router.ts

```diff
@@ -56,6 +56,6 @@ const batchRouter = Router()
  *        description: The API key
  *        example: '1c7ebe32457039cb2e98141e746e081d2a10282fd407e02e538ef72638955b08'
  */
-batchRouter.post('/', checkApiKey, BatchController.post.bind(BatchController))
+batchRouter.post('/', checkApiKey, BatchController.post.bind(BatchController));
```

Comment:
Add validation for raw list of raw review schema

Reply:
I implement validation using "zod" 

see:
validate batch request (still in progress) -> [dc89369](https://github.com/RevisarAI/RevisarAI/pull/63/commits/dc89369e899c73ce84e2b7f99d423dcae486938d)

but it not working yet, what I did wrong ? The zod package is new for me :(

I added to the push_revies.py a key "t" to make the request invalid and the zod say it valid

Reply:
You can add .strict(). When using parse extra fields are ok but if you want to remove it use the strict methods like so:
https://stackoverflow.com/questions/70415330/do-not-allow-extra-properties-with-zod-parse
`schemaValidationMiddleware({ body: IBatchReviewList.strict() })`